server/src/routers/api/userRouter.js

```diff
@@ -41,4 +48,15 @@ router.get("/", auth, (req, res) => {
   res.send(req.user);
 });
 
+// @route POST api/users/logout
+// @desc Logout a user
+// @access Private
+
+router.post("/logout", auth, async (req, res) => {
+  res.clearCookie("token");
+  res.status(200).json({
+    msg: "User Signout Successfully",
+  });
+});
+
```

Comment:
Also, please include screenshots in your PRs, 
There is almost a week left so hurry up.
You won't be able to get points after 20th Nov.